vms/proposervm/block/parse_test.go

```diff
@@ -179,6 +213,7 @@ func TestParseBytes(t *testing.T) {
 		expectedErr error
 	}{
 		{
+			// TODO: Include the epoch information in the serialized block
```

Comment:
Should this be addressed in this PR?

Reply:
removed the comment since explicit tests were added